### PR TITLE
10x Developers

### PR DESCRIPTION
**Pull request by adamgordonbell**

### CODE REVIEW

**Comment by jalletto** [comment]
Preview URL: https://6303ac8926a57525689eec3c--ephemeral-narwhal-3ac3b2.netlify.app/

File: blog/_posts/2022-08-17-stop-10x-developer.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+But how much day-to-day work has a high enough ceiling to let a skilled developer shine?
+
+## Doctor House, 10x Doctor
+
+> 10x engineers do not know what sunlight is because they have never seen it.
+>
+> They also hang from the roofs of caves, their leather wings enfolding their self assembled Compute as they code through the dark night of the soul...  
+> [Paul Sweeney](https://twitter.com/PaulSweeney/status/1150366063303057408)
+
+Variation in skills isn't unique to software development. If 10x developers exist, then 10x doctors and mechanics exist as well. But, for a skill to materially impact a team or an organization, it must be a limiting factor. If Dr. House worked at a walk-in clinic, handing out flu shots and antibiotics, he wouldn't be more effective than a mediocre doctor. Similarly, if your team's problems aren't occasionally [combinatorial graph coverage problems](https://leetcode.com/problems/minimum-weighted-subgraph-with-the-required-paths/submissions/), then is a Leetcode champ going to move the needle? If a 10x developer is supposed to get 10x the work done, then you need some challenging problems to throw at them.
```
Comment (jalletto) [inline blog/_posts/2022-08-17-stop-10x-developer.md:64]: Just a thought, everyone might not know who Dr. House is.

**Review by jalletto — APPROVED** [review]
Really great read!

File: blog/_posts/2022-08-17-stop-10x-developer.md (hunk 1 above)
Comment (adamgordonbell) [reply blog/_posts/2022-08-17-stop-10x-developer.md:64]: ah, good point. Will update

**Pull request merged**